program/Project Descriptions and Plans/SDK/sdk-project-concept.md

```diff
@@ -289,7 +295,7 @@ Unless **otherwise indicated in bold**, resource is not yet available. In summar
   - specification of the debugger interface;
   - hardware for testing;
   - 1 engineer month expertise in Eclipse CDT debugger integration;
-  - 1 engineer month exertise in Eclipse CDT third party tool integration (Symbiflow); and
+  - 1 engineer month expertise in Eclipse CDT third party tool integration (Symbiflow); and
   - 2 engineer months expertise in example integration within Eclipse CDT.
 
```

Comment:
It could require more depending on amount of boards that we want to support, the estimation should look like:
```
number_of_eng_months = X * number_of_boards + Y(level_of_productization)
```
   where `level_of_productization` is the grade of functionality from "simple extension for Eclipse CDT" to "usable product"

Reply:
Can you advise wording I should use?

Reply:
If we are talking about supporting one and only one board per SDK release - this comment can be omitted.